<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1855</width>
    <height>1013</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="capitulate_pushButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Сдаться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawRemain_pushButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>150</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Предложить ничью</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_1">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>740</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>350</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Соперник отправил предложение на ничью</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acceptDraw_pushButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>420</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Согласиться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="declineDraw_pushButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>420</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Отклонить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1855</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
